libs/mrwebrtc-unityplugin/src/native_renderer.cpp

```diff
@@ -133,45 +156,78 @@ void NativeRenderer::DisableRemoteVideo() {
 }
 
 void NativeRenderer::I420ARemoteVideoFrameCallback(
-    void* user_data,
-    const mrsI420AVideoFrame& frame) {
+  void* user_data,
+  const mrsI420AVideoFrame& frame) {
 
-    NativeRenderer* nativeVideo = static_cast<NativeRenderer*>(user_data);
+  // It is possible for one buffer to be empty, each buffer must be checked.
+  if (frame.ydata_ == nullptr || frame.udata_ == nullptr || frame.vdata_ == nullptr) return;
```

Comment:
Avoid single-line `if`, always use braces.
```suggestion
  if (frame.ydata_ == nullptr || frame.udata_ == nullptr || frame.vdata_ == nullptr) {
    return;
  }
```